frontend/components/GLBViewer.tsx

```
@@ -57,17 +57,15 @@ export default function GLBViewer({ src, camera }: Props) {
       <Canvas camera={cameraConfig}>
         <ambientLight intensity={0.8} />
         <Suspense
           fallback={
-            <Html
-              center
-              role="status"
-              style={{ padding: '0.75rem 1rem', background: 'rgba(0,0,0,0.6)', color: '#fff', borderRadius: '0.5rem' }}
-            >
-              Loading 3D model…
+            <Html center>
+              <output role="status" aria-live="polite" className="viewer-loading">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)